providers/apis/defi/curve/api_handler.go

```
@@ -0,0 +1,108 @@
+	}
+
+	return &APIHandler{
+		api:   api,
+		cache: types.NewProviderTickers(),
+	}, nil
+}
+
+func (h *APIHandler) CreateURL(
+	tickers []types.ProviderTicker,
+) (string, error) {
+	var addresses = make([]string, len(tickers))
```

golangci-lint (1.64.8)

File is not properly formatted

(gofumpt)